### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by alexpuente0**
On this Pull request, we:

- Added a new column for Species.
- Added new data to the table.
- Added Queries for data analysis.

  - Inside a transaction update the animals' table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
Q1

  - Update the animals' table by setting the species column to Digimon for all animals that have a name ending in mon and to pokemon for all animals that don't have species already set.
Q2

  - Delete all records in the animals' table, then roll back the transaction.
Q3

  - Delete all animals born after Jan 1st, 2022, Create a savepoint for the transaction, Update all animals' weight to be their weight multiplied by -1, Rollback to the savepoint, Update all animals' weights that are negative to be their weight multiplied by -1 and Commit the transaction.
Q4

  - How many animals are there?
q5

  - How many animals have never tried to escape?
q6

  - What is the average weight of animals?
q7

  - Who escapes the most, neutered or not neutered animals?
q8

  - What is the minimum and maximum weight of each type of animal?
q9

  - What is the average number of escape attempts per animal type of those born between 1990 and 2000?
q10

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,10 @@ SELECT (name, escape_attempts) FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg >= 10.4 and weight_kg <= 17.3;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, COUNT(escape_attempts) FROM animals GROUP BY neutered;
+SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
```
Comment (AnselemOdims) [inline queries.sql:46]: -  [x] Good job adding the queries but please include queries for the `UPDATE` and `DELETE` transactions as specified in the requirements.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @alexpuente0 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (alexpuente0) [reply queries.sql:46]: Of course! thanks for the feedback, I thought it was only for the analytics queries.

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @alexpuente0, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**